Github Resolver Comment (#44264)
demisto/content: Packs/PAN-OS/Integrations/Panorama/README.md

| PANOS.RuleHitCount.rule_modification_timestamp | Date | Timestamp when the rule was most recently modified. |
| PANOS.RuleHitCount.rulebase | String | The rulebase of the rule \(such as security, nat, and so on\). |
| PANOS.RuleHitCount.vsys | String | The name of the firewall VSYS. |

> 

Reviewer: ```suggestion
```

> ### pan-os-new-command

Author: ```suggestion
### pan-os-new-command-for-summery-demo
```

***

> A new command for demo only, remove after demo.

Author: ```suggestion
A NEW COMMAND FOR DEMO ONLY, REMOVE AFTER DEMO.
```

#### Base Command

`pan-os-new-command

> #### Input

Author: ```suggestion
#### Input Input Input Input
```

> | **Argument Name** | **Description** | **Required** |

Author: add a new column to this table, named "New argument" and mark all the rows aith the value "yes"

| --- | --- | --- |
| device_filter_string | The string by which to filter the results to only show specific hostnames or serial numbers. | Optional |
| target | The target number of the firewall. Used only on a Panorama instance. | Optional |
| rulebase | The firewall rulebase to check. Possible values are: application-override, authentication, decryption, dos, nat, network-packet-broker, pbf, qos, sdwan, security, tunnel-inspect. Default is security. | Optional |
| vsys | The firewall VSYS name to check.  Returns results for all VSYS if left blank. Default is all. | Optional |
| rules | Comma-separated list of rule names to check.  Returns results for all rules if left blank. Default is all. | Optional |
| unused_only | If set to true, only returns rules with a hit count of 0. Possible values are: true, false. Default is false. | Optional |

> | no_new_hits_since | Shows rules that have had hits, but not after the date provided (in the format YYYY/MM/DD HH:MM:SS). | Optional |

Reviewer: ```suggestion
| no_new_hits_since | Shows rules that have had hits, but not after the date provided (in the format YYYY-MM-DDTHH:MM:SSZ). | Optional |
```

> #### Context Output

Author: change this title to be UperCase
